lib/mgmt_be_client.c

```diff
@@ -731,23 +717,28 @@ static int mgmt_be_process_cfg_apply(struct mgmt_be_client *client_ctx,
 {
 	struct mgmt_be_txn_ctx *txn;
 
-	txn = mgmt_be_find_txn_by_id(client_ctx, txn_id);
-	if (!txn) {
-		mgmt_be_send_apply_reply(client_ctx, txn_id, NULL, 0, false,
-					    "Transaction not created yet!");
-		return -1;
-	}
+	txn = mgmt_be_find_txn_by_id(client_ctx, txn_id, true);
+	if (!txn)
+		goto failed;
 
 	MGMTD_BE_CLIENT_DBG("Trigger CFG_APPLY_REQ processing");
-	mgmt_be_txn_proc_cfgapply(txn);
+	if (mgmt_be_txn_proc_cfgapply(txn))
```

Comment:
`mgmt_be_txn_proc_cfgapply` always returns 0

Reply:
Let's merge this as-is I will make another pass to enable more error returns. It's not just this function that currently doesn't return all failures. It looks like `mgmt_be_send_cfgdata_create_reply` return is ignored most place, for example.